Bottom: 58d85caabb21531d21cbaae338d31045dbecad21
Top:    4ecf68c79b54229ab2d9666eff07ea1303946bda
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-04 14:55:08 +0800

Refresh of autofs-5.0.7-syncronize-handle_mounts-shutdown.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 59d4da2..3781ca3 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1356,7 +1356,7 @@ static void *statemachine(void *arg)
 			} else {
 				if (master_done(master_list)) {
 					master_mutex_unlock();
-					break;
+					return NULL;
 				}
 			}
 			master_mutex_unlock();
